### ISSUE TITLE
1.16 or 1.18 please [Suggestion]

### ISSUE THREAD

**Issue by AndrewthepikaDev** [GregTechCE/GregTech#1739]
Can you plz port to 1.16 or 1.18?

**Comment by Exaxxion** [comment]
Not really any plan to do this. The active development is mostly happening on the GTCEu fork currently which is strictly for 1.12.2. If you want a GregTech type mod for 1.18 you could always try [GregTech Intergalactical](https://www.curseforge.com/minecraft/mc-mods/gregtech-intergalactical)?

**Comment by warjort** [comment]
Or for the fabric loader there is https://www.curseforge.com/minecraft/mc-mods/modern-industrialization

**Comment by AndrewthepikaDev** [comment]
The Intergalactical one hasn't had any activity since April and the Fabric one is on Fabric and Fabric is trash.

**Comment by Rongmario** [comment]
There's no activity as I decided to stay on the glorious 1.12.2 version.

**Comment by AndrewthepikaDev** [comment]
But... 1.12.2 is losing support. And the textures on 1.14+ are INFINITELY better. Not to mention 1.16 and 1.18 have loads of new and interesting features you can integrate into GT somehow. I'm sure you can add a ton to ore generation with 1.18's extended world height, which goes from y-64 to y320. And 1.16 has loads of epic nether features like netherite, which has a fire-immune property that you can add for GT materials with a high melting point like iridium. Also I predominantly play 1.16 and 1.18 packs, along with many others I believe, so if you support the newer versions you will have more downloads. TLDR please update because 1.12.2 is getting outdated.

**Comment by brachy84** [comment]
Yes very epic

**Comment by AndrewthepikaDev** [comment]
1.16 is my favorite update if you couldn't tell
But seriously I think materials like iridium or osmium which have an extremely high melting point irl should be immune to fire and lava like netherite. And the 1.12.2 version really is losing support from devs, the current focused version is 1.18 with LTS 1.16.

**Comment by rhgngle** [comment]
1.12 is far from losing support, you just havent dug deep enough

**Comment by RehabCZ** [comment]
Still version 1.12.2 is very active and with community like [CleanroomMC](https://github.com/CleanroomMC) it will be for really long time.

**Comment by AndrewthepikaDev** [comment]
Ok. In 10 years when everyone plays 1.28 or something and 1.12 is as old as beta is today, I'll check back and see how these mods fared.

**Comment by serenibyss** [comment]
Glad we came to a conclusion

**Comment by AndrewthepikaDev** [comment]
anger noises

**Comment by bioastroiner** [comment]
a very big strong No, i suggest invest some time on copium or hire some devs to do it (prob won't still do it lol)
